.pre-commit-config.yaml

```diff
@@ -7,13 +7,16 @@ repos:
     hooks:
       - id: end-of-file-fixer
       - id: mixed-line-ending
-        args: [--fix=lf]
+        args:
+          - --fix=lf
       - id: trailing-whitespace
-        args: [--markdown-linebreak-ext=md]
+        args:
+          - --markdown-linebreak-ext=md
       # - id: check-executables-have-shebangs
```

Comment:
Also here

Reply:
im going to sort this after we have merged, will create a ticket for that